### PR TITLE
fix(@angular/cli): Add an error message when we call `ng doc` without…

### PR DESCRIPTION
**Pull request by wKoza**
… keyword. Currently, we open a page with the keyword `undefined`.

### CODE REVIEW

File: packages/angular/cli/commands/doc-impl.ts [hunk 1]
```diff
@@ -14,13 +14,18 @@ const opn = require('opn');
...
-      searchUrl = `https://www.google.com/search?q=site%3Aangular.io+${options.keyword}`;
-    }
+    if (options.keyword) {
+      let searchUrl = `https://angular.io/api?query=${options.keyword}`;
+      if (options.search) {
+        searchUrl = `https://www.google.com/search?q=site%3Aangular.io+${options.keyword}`;
+      }
 
-    return opn(searchUrl, {
+      return opn(searchUrl, {
```
Comment (mgechev) [inline packages/angular/cli/commands/doc-impl.ts]: The indentation doesn't look right. `wait: false` needs two more tabs?

Comment (wKoza) [reply packages/angular/cli/commands/doc-impl.ts]: 😱

File: packages/angular/cli/commands/doc-impl.ts [hunk 2]
```diff
@@ -14,13 +14,18 @@ const opn = require('opn');
 
 export class DocCommand extends Command<DocCommandSchema> {
   public async run(options: DocCommandSchema & Arguments) {
-    let searchUrl = `https://angular.io/api?query=${options.keyword}`;
-    if (options.search) {
-      searchUrl = `https://www.google.com/search?q=site%3Aangular.io+${options.keyword}`;
-    }
+    if (options.keyword) {
```
[the remark this answers is not in the document]
Comment (wKoza) [reply packages/angular/cli/commands/doc-impl.ts]: done

**Comment by mgechev** [comment]
Validate seem unhappy because of the commit message

**Comment by wKoza** [comment]
Yes, I've seen but I'm not well equipped `Windows` :wink:...

**Comment by wKoza** [comment]
validate doesn't like `--fixup` ? OK OK ...

**Pull request merged**